### PR TITLE
List courses

### LINKED ISSUE #60: Hawthorn: custom /courses page
As a member of the MIT community, I would like a more condensed list of courses that emphasize numbers and titles and not course images. 

We should apply a theme override to the /courses page and display the course listing like this:

image

The tricky part will be applying the expected sort of courses by department and the segregation of credit-bearing courses from non-credit courses, as detailed in https://goo.gl/kah2Br

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #60

#### What's this PR do?
Make courses appear in a list, separalety from non-credit courses.

screen shot 2018-04-04 at 4 46 52 pm

#### How should this be manually tested?
Run `make lms-static`, and go to the courses view.

### CODE REVIEW

File: lms/static/sass/_overrides.scss [hunk 1]
```diff
@@ -17,11 +17,45 @@ ol.nav-global>li>a {
...
+      .courses-listing .courses-listing-item {
+        width: 100% !important;
+        margin: 5px 0 10px !important;
+
+        a {
+          text-decoration: underline;
+        }
+
+        a:not(.btn) {
+          color: #003655 !important;
```
Comment (amir-qayyum-khan) [inline lms/static/sass/_overrides.scss]: can we declare a variable for it?

File: lms/templates/course.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      else:
+          course_date_string = ''
+      %>
+      % if course.advertised_start is not None:
+          <div class="course-date" aria-hidden="true">${_("Starts")}: ${course.advertised_start}</div>
+      % else:
+          <div class="course-date localized_datetime" aria-hidden="true" data-format="shortDate" data-datetime="${course_date_string}" data-string="${_("Starts: {date}")}"></div>
+      % endif
+    </div>
+    <div class="sr">
```
Comment (amir-qayyum-khan) [inline lms/templates/course.html]: you converted table like structure to list for accessible?

Comment (amir-qayyum-khan) [reply lms/templates/course.html]: why not use list from start and remove this section at all. You can style it to display horizontal?

**Comment by annagav** [comment]
Maybe this link color is better:

screen shot 2018-04-05 at 2 00 53 pm

File: lms/templates/course.html (hunk 2 above)
Comment (annagav) [reply lms/templates/course.html]: I removed that section

**Comment by pdpinch** [comment]
I spoke with Tobias about deploying this to mitx-qa-next. I'll review it there with a (nearly) production list of courses

**Pull request merged**